.ci/eclipse-compiler-javac.sh

```diff
@@ -1,16 +1,23 @@
 #!/bin/sh
-set -e
+
+ECJ_JAR="ecj-4.7.jar"
+ECJ_MAVEN_VERSION="R-4.7-201706120950"
 
 ECJ_VERSION="4.7"
 mkdir -p target/classes
 mkdir -p target/eclipse
 
-if [ ! -e ~/.m2/repository/ecj-$ECJ_VERSION.jar ]; then
-    wget http://ftp-stud.fht-esslingen.de/pub/Mirrors/eclipse/eclipse/downloads/drops4/R-4.7-201706120950/"ecj-$ECJ_VERSION.jar" -P ~/.m2/repository/
+if [ ! -e ~/.m2/repository/$ECJ_JAR ]; then
```

Comment:
lets keep our copy in the same folder as on server - "~/.m2/repository/$ECJ_MAVEN_VERSION/$ECJ_JAR"

> `! -e`

should be `! -f `